catalyst_voices/lib/common/ext/string_ext.dart

```diff
@@ -0,0 +1,5 @@
+extension StringExt on String {
+  String capitalize() {
+    return '${this[0].toUpperCase()}${substring(1).toLowerCase()}';
```

Comment:
Please handle empty and single character length strings. It's an edge case but it's not too complicated that we couldn't do it straight away